suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -479,7 +479,7 @@ export const networks = {
             },
         },
         coingeckoId: undefined,
-        coingeckoNativeId: undefined,
+        coingeckoNativeId: 'test-bitcoin', // fake, coingecko does not have testnets
```

Comment:
Is it possible to upload our own bitcoin/ethereum/... testnet icons to `data.trezor.io`? If not, please uppercase the placeholder letter
![Screenshot 2024-09-30 at 11 23 32](https://github.com/user-attachments/assets/a88a8d2f-778a-45d5-967e-a0880cc62b9a)

Reply:
1. fixed in https://github.com/trezor/trezor-suite/pull/14217/commits/d0e09693587bc4c82b1963aab2759159fff11686
2. fixed in https://github.com/trezor/trezor-suite/pull/14217/commits/d0e09693587bc4c82b1963aab2759159fff11686
3. it is possible to upload custom icons, I would create a separate issue for it; for now I changed the placeholder to uppercase